docs/0_internal/4_a_pause_contracts.md

```diff
@@ -0,0 +1,75 @@
+# Pause Contracts
+This doc contains a guide on how to use the Pausable functionality of Aligned.
+
+To run the make targets specified in this guide, you must first have the relevant following env vars under `contracts/scripts/.env`:
+```
+export RPC_URL=<rpc_url>
+export ALIGNED_SERVICE_MANAGER=<aligned_contract_address>
+export ALIGNED_SERVICE_MANAGER_PAUSER_PRIVATE_KEY=<aligned_service_manager_pauser_private_key>
+export BATCHER_PAYMENT_SERVICE=<payment_service_contract_address>
+export BATCHER_PAYMENT_SERVICE_PAUSER_PRIVATE_KEY=<batcher_payment_service_pauser_private_key>
+```
+
+## Aligned Service Manager
+
+Aligned Service Manager is granulary pausable, which means you can pause the whole contract, or only specific functions. For this,
+Aligned uses the Pauser Registry contract provided by Eigenlayer. This contract stores the role of different accounts, so
+you can have X pausers and Y unpausers.
+
+To interact with it you can:
+
+- Get current paused state:
+```
+make get_paused_state_aligned_service_manager
+```
```

Comment:
`source contracts/scripts/.env` is needed before running all targets in this doc

Reply:
wip fix

Reply:
fixed on https://github.com/yetanotherco/aligned_layer/pull/1501